integration-webcert/src/main/java/se/inera/intyg/minaintyg/integration/webcert/client/PrintCertificateFromWebcertService.java

```diff
@@ -0,0 +1,48 @@
+package se.inera.intyg.minaintyg.integration.webcert.client;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClient;
+import se.inera.intyg.minaintyg.integration.api.certificate.PrintCertificateIntegrationRequest;
+import se.inera.intyg.minaintyg.integration.webcert.client.dto.PrintCertificateResponseDTO;
+
+@Service
+public class PrintCertificateFromWebcertService {
+
+  private final WebClient webClient;
+  private final String scheme;
+  private final String baseUrl;
+  private final int port;
+  private final String endpoint;
+
+  public PrintCertificateFromWebcertService(
+      @Qualifier(value = "webcertWebClient") WebClient webClient,
+      @Value("${integration.webcert.scheme}") String scheme,
+      @Value("${integration.webcert.baseurl}") String baseUrl,
+      @Value("${integration.webcert.port}") int port,
+      @Value("${integration.webcert.printcertificate.endpoint}") String endpoint) {
+    this.webClient = webClient;
+    this.scheme = scheme;
+    this.baseUrl = baseUrl;
+    this.port = port;
+    this.endpoint = endpoint;
+
+  }
+
+  public PrintCertificateResponseDTO print(PrintCertificateIntegrationRequest request) {
+    return webClient.get().uri(uriBuilder -> uriBuilder
```

Comment:
Should this be a GET or should it be a POST? Isn't certificateId sent as path parameter? Maybe we should sync this with Webcert.

Reply:
Fixed